themes/hive/theme.toml

```diff
@@ -0,0 +1,21 @@
+# theme.toml template for a Hugo theme
+# See https://github.com/gohugoio/hugoThemes#themetoml for an example
+
+name = "Hive"
+license = "APACHE LICENSE, VERSION 2.0"
+licenselink = "https://www.apache.org/licenses/LICENSE-2.0.html"
+description = ""
+homepage = "https://hive.apache.org/"
+tags = []
+features = []
+min_version = "0.41.0"
+
+[author]
+  name = "simhadri-g"
+  homepage = ""
```

Comment:
Author tags are usually not present in Apache code so not sure if we should/can include this.

Reply:
Ok, I will change the name to "hive".